users/syncing.rst

```diff
@@ -79,6 +79,38 @@ synchronize the file. The block lists are compared to build a list of needed
 blocks, which are then requested from the network or copied locally, as
 described above.
 
+.. _case-sensitivity:
+
+Case Sensitivity in File Names
+------------------------------
+
+In principle Syncthing works with *case-sensitive* paths, meaning that
+`file.txt` and `FILE.txt` denote two independent things. It does not
+support synchronizing the contents of both as if they were somehow
+related.
+
+Some operating systems (e.g. Windows, Mac and Android) assume the
+opposite, treating them as the same file (or directory) and can never
+have both file names simultaneously. They will *preserve* the case
+from file creation, however.
```

Comment:
I wouldn't talk about what it "supports" or not, that sounds as if this was about feature completeness, when it is a fundamental design choice.

Also I am pretty sure Android is case-sensitive. External SDs used to be FAT, which is case-insensitive, but as far as I remember that hasn't been in use after Android 4.

Last point: What does preserve actually mean in this context? I understand what it means when syncing, but an OS/FS doesn't do that. Likely depending on what it means, is it relevant here? The main point is the sentence before, that on such systems no two files with names differing in case only can exist. I general I'd leave out any information that does not directly help understanding the point to make, otherwise it's just potential for distraction or even confusion.

```suggestion
Syncthing works with *case-sensitive* paths, meaning that
`file.txt` and `FILE.txt` denote two independent files. Some operating systems (e.g. Windows, Mac) or filesystems (e.g. FAT) assume the
opposite, treating them as the same file (or directory) and can never
have both file names simultaneously. They will *preserve* the case
from file creation, however.
```

Reply:
> Also I am pretty sure Android is case-sensitive. External SDs used to be FAT, which is case-insensitive, but as far as I remember that hasn't been in use after Android 4.

SD cards in Android are still (ex)FAT, and thus case insensitive, as far as I know. This is because they are designed to be ejected and used in different devices, which includes usually Windows, and Windows by default does not support any Linux file systems.

Reply:
> I wouldn't talk about what it "supports" or not, that sounds as if this was about feature completeness, when it is a fundamental design choice.

Stating that it it doesn't support any other view of the world is trying to preempt questions about being able to change that to what many users think is normal coming from the Windows world. I still think it's important to clear up the two names never being considered related.

> Also I am pretty sure Android is case-sensitive. External SDs used to be FAT, which is case-insensitive, but as far as I remember that hasn't been in use after Android 4.

At least on my Android 7.1 device, the file manager will refuse to create a case-conflicting file. The "internal" storage is mounted as fstype `sdcardfs`, whatever that means. I'm not an expert on Android, so this information was taken from some maintainer's statement on an issue or forum discussion - can't find it quickly right now.

> Last point: What does preserve actually mean in this context? I understand what it means when syncing, but an OS/FS doesn't do that. Likely depending on what it means, is it relevant here? The main point is the sentence before, that on such systems no two files with names differing in case only can exist. I general I'd leave out any information that does not directly help understanding the point to make, otherwise it's just potential for distraction or even confusion.

Trying to bring in the technical terminology "case-preserving" for those who might have read about different "levels" of case-sensitivity after growing up with MS-DOS :-) You're right though, it's not really needed here.